Issue of Minecraft Crashing on Startup after adding the mod to my instance in curseforge 1.21.10 Fabric

---- Minecraft Crash Report ----
// There are four lights!

Time: 2025-12-13 15:45:46
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'weatherchanger' at 'me.lucaslah.weatherchanger.fabric.WeatherChangerFabric'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$0(FabricLoaderImpl.java:409)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:407)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.class_310.<init>(class_310.java:483)
	at knot//net.minecraft.client.main.Main.main(Main.java:234)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NoSuchMethodError: 'void net.minecraft.class_304.<init>(java.lang.String, net.minecraft.class_3675$class_307, int, java.lang.String)'
	at knot//me.lucaslah.weatherchanger.keybinding.Key.<init>(Key.java:14)
	at knot//me.lucaslah.weatherchanger.keys.ToggleClearKey.<init>(ToggleClearKey.java:16)
	at knot//me.lucaslah.weatherchanger.WeatherChanger.init(WeatherChanger.java:56)
	at knot//me.lucaslah.weatherchanger.fabric.WeatherChangerFabric.onInitializeClient(WeatherChangerFabric.java:16)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:405)
	... 6 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$0(FabricLoaderImpl.java:409)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:407)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.class_310.<init>(class_310.java:483)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Erweiterte Windows 32 Base-API:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:Bibliothek für Steuerelemente:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Krypto-API32:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.26100.5074 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS-Client-API-DLL:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP-Hilfs-API:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Client-DLL für Windows NT-Basis-API:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Client-DLL für Windows NT-Basis-API:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		MpOav.dll:IOfficeAntiVirus Module:4.18.25110.5 (4a6978a838df7287d907bb2ee8a2eeb2373b186e):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE für Windows:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		POWRPROF.dll:Power Profile Helper DLL:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Unterstützungs-DLL für Leistungsdaten:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remoteprozeduraufruf-Laufzeitumgebung:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Allgemeine Windows-Shell-DLL:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Client-DLL für Windows USER-API (mehrere Benutzer):10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.38.33135.0:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.26100.1150 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP-Dienste:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API-DLL:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0-32-Bit-DLL:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Bibliothek mit kryptografischen Primitiven von Windows:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM für Windows:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		extnet.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		fwpuclnt.dll:FWP/IPsec Benutzermodus-API:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		java.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		javaw.exe:OpenJDK Platform binary:21.0.7.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		jli.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		jna13769283359003129889.dll:JNA native library:7.0.2:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:21.0.7.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		msvcp140.dll:Microsoft® C Runtime Library:14.38.33135.0:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0-Dienstanbieter:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-Mail-Namenshimanbieter:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt-Router:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		nio.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:DLL für NT-Layer:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:DLL für Windows-Systemleistungsobjekte:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.26100.4202 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.26100.7171 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight-Hilfsprogrammbibliothek:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.38.33135.0:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
		win32u.dll:Win32u:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage-API:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows-Basistypen-DLL:10.0.26100.7306 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.26100.5074 (WinBuild.160101.0800):Microsoft Corporation
		wshunix.dll:AF_UNIX Winsock2 Helper DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:21.0.7.0:Microsoft
Stacktrace:
	at knot//net.minecraft.client.main.Main.main(Main.java:234)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.21.10
	Minecraft Version ID: 1.21.10
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.7, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 174880080 bytes (166 MiB) / 520093696 bytes (496 MiB) up to 12884901888 bytes (12288 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 7800X3D 8-Core Processor           
	Identifier: AuthenticAMD Family 25 Model 97 Stepping 2
	Microarchitecture: Zen 3
	Frequency (GHz): 4.19
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: AMD Radeon RX 7900 XTX
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MiB): 24560.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 32.0.22029.9039
	Graphics card #1 name: AMD Radeon(TM) Graphics
	Graphics card #1 vendor: Advanced Micro Devices, Inc.
	Graphics card #1 VRAM (MiB): 512.00
	Graphics card #1 deviceId: VideoController2
	Graphics card #1 versionInfo: 32.0.21033.2001
	Memory slot #0 capacity (MiB): 16384.00
	Memory slot #0 clockSpeed (GHz): 6.00
	Memory slot #0 type: DDR5
	Memory slot #1 capacity (MiB): 16384.00
	Memory slot #1 clockSpeed (GHz): 6.00
	Memory slot #1 type: DDR5
	Virtual memory max (MiB): 78999.14
	Virtual memory used (MiB): 33288.65
	Swap memory total (MiB): 47104.00
	Swap memory used (MiB): 162.21
	Space in storage for jna.tmpdir (MiB): available: 153305.16, total: 976645.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 153305.16, total: 976645.00
	Space in storage for io.netty.native.workdir (MiB): available: 153305.16, total: 976645.00
	Space in storage for java.io.tmpdir (MiB): available: 153305.16, total: 976645.00
	Space in storage for workdir (MiB): available: 153305.16, total: 976645.00
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx12288m -Xms256m
	Debug Flags: 0 total; 
	Fabric Mods: 
		appleskin: AppleSkin 3.0.7+mc1.21.9
		clientsort: ClientSort 2.1.4+1.21.10
		cloth-config: Cloth Config v20 20.0.149
			cloth-basic-math: cloth-basic-math 0.6.1
		enchdesc: EnchantmentDescriptions 21.10.1
		entity_model_features: Entity Model Features 3.0.7
		entity_texture_features: Entity Texture Features 7.0.6
			org_apache_httpcomponents_httpmime: httpmime 4.5.10
		fabric-api: Fabric API 0.138.3+1.21.10
			fabric-api-base: Fabric API Base 1.0.0+14b92d896f
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.106+d17682156f
			fabric-biome-api-v1: Fabric Biome API (v1) 16.1.0+c2417d4f6f
			fabric-block-api-v1: Fabric Block API (v1) 1.1.5+d17682156f
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.34+e4fd1b926f
			fabric-command-api-v2: Fabric Command API (v2) 2.4.0+c0ab2d5d6f
			fabric-content-registries-v0: Fabric Content Registries (v0) 10.2.6+b6a242646f
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.49+7f945d5b6f
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.17.0+ff7bc77d6f
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.18+33df5e6e6f
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.8.33+cd3b9e596f
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 23.2.27+d8b85ef16f
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.22+38b9cc196f
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.1.13+fde3702c6f
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 4.0.36+5ba217126f
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.1.1+e47eaae56f
			fabric-item-api-v1: Fabric Item API (v1) 11.5.11+9bce411b6f
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.2.27+86b3ae576f
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.1.1+b3ba9c486f
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.6.9+33df5e6e6f
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.67+3f89f5a56f
			fabric-loot-api-v3: Fabric Loot API (v3) 2.0.14+d17682156f
			fabric-message-api-v1: Fabric Message API (v1) 6.1.7+3154c1c86f
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 6.0.4+3ffd6ef06f
			fabric-networking-api-v1: Fabric Networking API (v1) 5.1.0+b095d8db6f
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 21.1.28+9bce411b6f
			fabric-particles-v1: Fabric Particles (v1) 4.2.3+a0ee59a46f
			fabric-recipe-api-v1: Fabric Recipe API (v1) 8.2.0+39709ef06f
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 6.1.42+679c8c5c6f
			fabric-renderer-api-v1: Fabric Renderer API (v1) 7.2.2+3706cdf36f
			fabric-renderer-indigo: Fabric Renderer - Indigo 4.1.3+d7df91576f
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.38+86b3ae576f
			fabric-rendering-v1: Fabric Rendering (v1) 16.2.0+bee81f016f
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.28+33df5e6e6f
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.3.1+d4d4122a6f
			fabric-resource-loader-v1: Fabric Resource Loader (v1) 1.0.4+14959b836f
			fabric-screen-api-v1: Fabric Screen API (v1) 3.1.0+9c82a7986f
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.151+d17682156f
			fabric-serialization-api-v1: Fabric Serialization API (v1) 1.0.0+d857c9766f
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.45+328a75ba6f
			fabric-tag-api-v1: Fabric Tag API (v1) 1.2.13+2fa446b26f
			fabric-transfer-api-v1: Fabric Transfer API (v1) 6.0.15+79ec2c126f
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.5.3+41eb43436f
		fabricloader: Fabric Loader 0.18.1
			mixinextras: MixinExtras 0.5.0
		fusion: Fusion 1.2.11
		iris: Iris 1.9.6+mc1.21.9
			io_github_douira_glsl-transformer: glsl-transformer 3.0.0-pre3
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.13.1
		java: OpenJDK 64-Bit Server VM 21
		justzoom: Just Zoom 2.1.2
		konkrete: Konkrete 1.9.13
		minecraft: Minecraft 1.21.10
		mousetweaks: Mouse Tweaks 2.29
		placeholder-api: Placeholder API 2.8.1+1.21.10
		polytone: Polytone 1.21.10-3.5.1
			net_objecthunter_exp4j: exp4j 0.4.8
		prickle: PrickleMC 21.10.2
		seedcrackerx: SeedCrackerX 2.15.5
			com_seedfinding_latticg: latticg 1.7
			com_seedfinding_mc_biome: mc_biome 17af8cb1110fdc983b7cb2b887d1fb2060e23ee3
			com_seedfinding_mc_core: mc_core eee662999e9f3fe037476b6940dbd6d5e23cdbb6
			com_seedfinding_mc_feature: mc_feature 755d3611ac1c499c28289ccca5b738af6a5859b7
			com_seedfinding_mc_math: mc_math 851e9d0577dfdca50154e98f1d334bd31c641326
			com_seedfinding_mc_noise: mc_noise dbab3996ea3abff5dd420db53c31d5498afd2fe5
			com_seedfinding_mc_reversal: mc_reversal 75aa6ce47a9f53a1aa212765e9830e08f6c86299
			com_seedfinding_mc_seed: mc_seed 55f6242001f7eb4226df4ed0d023f1838a54a99d
			com_seedfinding_mc_terrain: mc_terrain a03e440ec5b282e399382f2cc5ad0db91b438d2e
		sodium: Sodium 0.7.3+mc1.21.10
		weatherchanger: WeatherChanger 1.2.2
		xaerominimap: Xaero's Minimap 25.2.15
		xaeroworldmap: Xaero's World Map 1.39.17
	Launched Version: fabric-loader-0.18.1-1.21.10
	Launcher name: minecraft-launcher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: Unknown
	Window size: <not initialized>
	GFLW Platform: <error>
	Render Extensions: ERR
	GL debug messages: <no renderer available>
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 404
	Type: Client (map_client.txt)
	Locale: en_US
	System encoding: Cp1252
	File encoding: UTF-8
	CPU: 16x AMD Ryzen 7 7800X3D 8-Core Processor

--- a/README.md
+++ b/README.md
@@ -58,8 +58,8 @@ See the [GitHub releases](https://github.com/Lucaslah/WeatherChanger/releases) p
 
 | Weather Changer Version | Minecraft Version | Mod loaders   | Latest Release                                                                     |
 |-------------------------|-------------------|---------------|------------------------------------------------------------------------------------|
-| v1.3.x                  | 1.21.11           | Fabric, Forge | [1.3.0-beta1](https://github.com/Lucaslah/WeatherChanger/releases/tag/1.3.0-beta1) |
-| v1.2.x                  | 1.21-1.21.10      | Fabric, Forge | [1.2.2](https://github.com/Lucaslah/WeatherChanger/releases/tag/1.2.2)             |
+| v1.3.x                  | 1.21.9-1.21.11    | Fabric, Forge | [1.3.0-beta1](https://github.com/Lucaslah/WeatherChanger/releases/tag/1.3.0-beta1) |
+| v1.2.x                  | 1.21-1.21.8       | Fabric, Forge | [1.2.2](https://github.com/Lucaslah/WeatherChanger/releases/tag/1.2.2)             |
 
 ------------------------------------------
 *Licensed under the [GNU Lesser General Public License v3.0](https://www.gnu.org/licenses/lgpl-3.0.en.html) license.*

